### PR TITLE
Generalize FGD Formatting

### PR DESCRIPTION
**Pull request by tmob03**
Formatting based on the trigger_momentum_X brush entities, let me know if some other style is most desirable and I will change this to that style.

### CODE REVIEW

File: fgd/brush/func/func_monitor.fgd [hunk 1]
```diff
@@ -1,5 +1,6 @@
-@SolidClass appliesto(+HL2_ENTITIES)
-		base(func_brush)
+@SolidClass
+	appliesto(+HL2_ENTITIES)
+	base(func_brush)
```
Comment (braem) [inline fgd/brush/func/func_monitor.fgd:3]: I think bases should always come first on same line. appliesto right after

File: fgd/brush/func/func_tank_combine_cannon.fgd [hunk 2]
```diff
@@ -1,5 +1,5 @@
 @SolidClass appliesto(+HL2_ENTITIES)
-		base(BaseTank) 
+	base(BaseTank)
```
Comment (braem) [inline fgd/brush/func/func_tank_combine_cannon.fgd:2]: Yeah same here and in the other few places

File: fgd/point/game/game_end.fgd [hunk 3]
```diff
@@ -1,5 +1,5 @@
-@PointClass appliesto(+USE_MULTIPLAYER)
-		base(BaseEntityPoint, MasterEnt)
+@PointClass base(BaseEntityPoint, MasterEnt)
+	appliesto(+USE_MULTIPLAYER)
```
Comment (braem) [inline fgd/point/game/game_end.fgd:2]: Yeah like this!

File: fgd/point/light/light_spot.fgd [hunk 4]
```diff
@@ -7,7 +7,7 @@
 	[
 	target(target_destination) : "Entity to point at" : : "The name of an entity in the map that the spotlight will point at. " +
 		"This will override the spotlight's angles."
-
+
```
Comment (braem) [inline fgd/point/light/light_spot.fgd:10]: (nit) shouldn't have any empty tabs

**Review by braem — CHANGES_REQUESTED** [review]
Wow! Thank you thank you thank you :D This has always bugged the hell outta me but not enough to take the time and do it. Makes me want a linter - could be a neat side project for ya if interested in code stuffs :o 

Only qualms I have is that base should be come before appliesto, but only a few occurrences of that really :)
Though I'd like to ask the changes to be in a new commit so review is easier; big change!

**Comment by braem** [comment]
@tmob03 did you mean to commit something? I see the review is marked resolved with no commit

**Comment by tmob03** [comment]
> @tmob03 did you mean to commit something? I see the review is marked resolved with no commit

Aw crap, thanks for telling me. I made the changes but forgot to commit before I went out of town

**Review by braem — APPROVED** [review]

**Comment by braem** [comment]
> Aw crap, thanks for telling me. I made the changes but forgot to commit before I went out of town

No worries haha. Exactly what I had thought happened :p

> Would def be neat to have one, I'm not smart enough to make it though 🙃

You are :)

**Pull request merged**